po/common/common.pot

```diff
@@ -810,6 +810,16 @@ msgctxt "footer_wiki_link"
 msgid "https://en.wiki.openfoodfacts.org"
 msgstr ""
 
+# Do not translate
```

Comment:
Do not translate "Open Beauty Facts" but do translate "Personal care"... I think "Cosmetics" would be safer, personal care can be too easily mistranslated. "Soin personnel" wouldn't mean much in French for instance.

Reply:
Ok for cosmetics. Anectodal evidence shows me that cosmetics looses us 50% of the potential userbase.